### PR TITLE
Fix recommended json file

### PR DESCRIPTION
**Pull request by estringana**
### Description
Move back to having the `recommended.json` file on the repo

### Reviewer checklist
- [ ] Test coverage seems ok.
- [ ] Appropriate labels assigned.

### CODE REVIEW

**Review by realFlowControl — APPROVED** [review]
Whoop Whoop 🎉

File: appsec/third_party/CMakeLists.txt [hunk 1]
```diff
@@ -59,13 +59,3 @@ endif()
...
 target_include_directories(zlibstatic INTERFACE ${zlib_SOURCE_DIR} ${zlib_BINARY_DIR})
 
-include(ExternalProject)
-ExternalProject_Add(event_rules
-    GIT_REPOSITORY https://github.com/DataDog/appsec-event-rules.git
-    GIT_TAG 1.9.0
-    GIT_SHALLOW ON
-    CONFIGURE_COMMAND ""
-    BUILD_COMMAND ""
-    INSTALL_COMMAND "")
```
Comment (realFlowControl) [inline appsec/third_party/CMakeLists.txt:69]: Just out of curiosity: if you do not download the rules file from there anymore, do you have another process in place to keep that updated?

**Review by realFlowControl — APPROVED** [review]

File: appsec/third_party/CMakeLists.txt (hunk 1 above)
Comment (estringana) [reply appsec/third_party/CMakeLists.txt:69]: It used to be like this. We had a local copy on the repository which was manually updated. When @Anilm3 moved appsec to this repo, he changed this. I had a with @cataphract yesterday and we both think it's more controller to have a local copy than downloading on the building process.

File: .circleci/continue_config.yml [hunk 2]
```diff
@@ -2841,11 +2841,9 @@ jobs:
             make -j $(nproc) ddappsec_helper_test
             ./tests/helper/ddappsec_helper_test
       - run:
-          name: Download ruleset
+          name: Copy ruleset
           command: |
-            cd appsec/build
-            make event_rules
-            cp ./third_party/event_rules-prefix/src/event_rules/build/recommended.json ../../appsec_$(uname -m)/
+            cp recommended.json ../appsec_$(uname -m)/
```
Comment (realFlowControl) [inline .circleci/continue_config.yml:2846]: Looks like something is wrong with the path here

Comment (estringana) [reply .circleci/continue_config.yml:2847]: Yes! I've just fixed it

**Pull request merged**